tests/test_routes.py

```diff
@@ -334,6 +339,77 @@ def test_method_not_allowed(self):
         response = self.client.put(f"{BASE_URL}", json=test_record.serialize())
         self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
 
+######################################################################
+    # T E S T   Q U E R Y   S T R I N G S
+######################################################################
+
+    def test_query_inventories_by_name(self):
+        """It should Query Inventories by Name"""
+        records = self._create_inventory_records(10)
+        test_name = records[0].name
+        name_list = [record for record in records if record.name == test_name]
+        logging.info("Name=%s: %d = %s", test_name, len(name_list), name_list)
+        resp = self.client.get(BASE_URL, query_string=f"name={quote_plus(test_name)}")
```

Comment:
What does `quote_plus` do?

Reply:
its used to encode a plus sign (which signifies a space) in the url